pkg/destroy/gcp/network.go

```diff
@@ -434,41 +531,56 @@ func (o *ClusterUninstaller) listNetworks() ([]nameAndURL, error) {
 }
 
 func (o *ClusterUninstaller) deleteNetwork(name string) error {
-	o.Logger.Debugf("Deleting network %s", name)
 	ctx, cancel := o.contextWithTimeout()
 	defer cancel()
-	_, err := o.computeSvc.Networks.Delete(o.ProjectID, name).Context(ctx).Do()
+
+	o.Logger.Debugf("Deleting network %s", name)
+	resp, err := o.computeSvc.Networks.Delete(o.ProjectID, name).RequestId(o.requestID("network", name)).Context(ctx).Do()
```

Comment:
most of the delete calls return an operation. so maybe this should be `op` instead of `resp`

Reply:
ack, will update